field_records/ubna_2023.csv

```diff
@@ -41,9 +41,15 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
\ No newline at end of file
+2023-07-11T15:41:00 , 2023-07-17T15:41:00 , H , 006 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 48000 , Medium , None , None , 4.098 , 3.796 , CT , MB , CT , recover-20230717 , Panasonic-Ikea Batteries Daytime 0:00-24:00 UTC; Reconfigured for Peace of Mind; Half and Half batteries since forgot a batch in APL
+2023-07-11T15:41:00 , 2023-07-17T15:41:00 , B , 004 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 192000 , Medium , None , None , 4.056 , 3.740 , CT , MB , CT , recover-20230717 , Panasonic Batteries Nighttime 3:06-13:23 UTC; Reconfigured Recording Period
+2023-07-11T15:25:00 , 2023-07-17T15:24:00 , G , 007 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 192000 , Medium , None , None , 4.078 , 3.753 , MB , MB , CT , recover-20230717 , Panasonic Batteries Nighttime 3:06-13:23 UTC; Reconfigured Recording Period
+2023-07-11T15:25:00 , 2023-07-17T15:24:00 , D , 008 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 48000 , Medium , None , None , 4.097 , 3.804 , MB , CT , CT , recover-20230717 , Ikea Batteries Daytime 0:00-24:00 UTC; Reconfigured for Peace of Mind
+2023-07-11T15:09:00 , 2023-07-17T15:01:00 , E , 009 , Central Pond , 47° 39' 15.762'' N , 122° 18' 1.992'' W , 192000 , Medium , None , None , 4.090 , 3.833 , CT , MB , CT , recover-20230717 , Ikea Batteries Nighttime 3:06-13:23 UTC; Reconfigured Recording Period
+2023-07-11T15:09:00 , 2023-07-17T15:01:00 , F , 012 , Central Pond , 47° 39' 15.762'' N , 122° 18' 1.992'' W , 48000 , Medium , None , None , 4.062 , 3.750 , CT , MB , CT , recover-20230717 , Panasonic Batteries Daytime 0:00-24:00 UTC; Reconfigured for Peace of Mind - Central Pond is noticably lower
+2023-07-17T15:47:00 , (DATE-TIME) , H , 005 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 48000 , Medium , None , None , 4.083 , (VOLTAGE) , CT , MB , MB , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
+2023-07-17T15:47:00 , (DATE-TIME) , B , 003 , Telephone Field , 47° 39' 31.122'' N , 122° 17' 18.132'' W , 192000 , Medium , None , None , 4.075 , (VOLTAGE) , CT , MB , MB , (RECOVER-DATE) , Panasonic Batteries Nighttime 3:06-13:23 UTC;
+2023-07-17T15:29:00 , (DATE-TIME) , G , 002 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 192000 , Medium , None , None , 4.024 , (VOLTAGE) , MB , CT , MB , (RECOVER-DATE) , Panasonic Batteries Nighttime 3:06-13:23 UTC;
+2023-07-17T15:29:00 , (DATE-TIME) , D , 010 , Foliage , 47° 39' 19.440'' N , 122° 17' 26.340'' W , 48000 , Medium , None , None , 4.087 , (VOLTAGE) , MB , CT , MB , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
+2023-07-17T15:12:00 , (DATE-TIME) , E , 001 , Central Pond , 47° 39' 15.762'' N , 122° 18' 1.992'' W , 192000 , Medium , None , None , 4.084 , (VOLTAGE) , CT , CT , MB , (RECOVER-DATE) , Ikea Batteries Nighttime 3:06-13:23 UTC; Central Pond is super low (ankle height at deployment site);
```

Comment:
> Central Pond is super low (ankle height at deployment site);

Good note here! @m-ball3 I think in your research you've come across how bats depend on water availability no? I'd love to find out if these recessions have any effects on activity!

Reply:
I did read something about this briefly! I will look further into it when I get a chance because I am also really interested! (: